OrlandoDiaz_Project1

od397 opened the pull request:
The beginning of my Project is just the sketch of the character I’ll be
using throughout the project. Look up “Bill Cipher” on Google for a
picture of the character in action in the cartoon. I tried to replicate
him the best I could. (Still working on the limbs though which are the
most difficult part)

od397 committed:
OrlandoDiaz_Project1

The beginning of my Project is just the sketch of the character I’ll be
using throughout the project. Look up “Bill Cipher” on Google for a
picture of the character in action in the cartoon. I tried to replicate
him the best I could. (Still working on the limbs though which are the
most difficult part)

LadyK commented:
This is a perfect piece of code to break into functions, before moving forward. How could you make more than one Bill Cipher appear? Also, what's the adjective you are portraying? After creating functions, think about how your are going to show and adjective. Keep it coming!

od397 commented:
Hi! and yeah I should practice on cloning him, and the adjective is
Untrustworthy (which is how he acts on the show) I was thinking of
portraying him changing his color, making his arms pose and flame up,
floating, and have the background change color all when the mouse is
pressed (although I might make him float forever). I want to show him
change from trusting to showing his "true colors". Although I'm not sure if
I can manage all those changes

On Tue, Sep 22, 2015 at 8:18 PM, LadyK notifications@github.com wrote:

> This is a perfect piece of code to break into functions, before moving
> forward. How could you make more than one Bill Cipher appear? Also, what's
> the adjective you are portraying? After creating functions, think about how
> your are going to show and adjective. Keep it coming!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/IDMUY1133BCreativeCodingAssignments/Project_1_Adjective/pull/4#issuecomment-142460648
> .

LadyK commented:
Sure you can! Just break it down into steps, focusing on a few things to achieve in each step. Make sure to use comments to show what you are trying to do.

od397 committed:
OrlandoDiaz_Project1

Update! Adjective is Untrustworthy

I added special events and text for when you start the program, press
“X”, and click on Bill Cipher.

od397 committed:
Adjective Project 1 Update

I added a “jumpscare” of sorts when you press “I/i” (get it? :P) I also
commented out a couple of lines that I tried to make Bill float (as in
move up and down) with. I had trouble finding out how and I elaborate
in the comments.

LadyK commented:
Hi Orlando! In order to make him float, you are going to have to do one of two methods: add a variable to _all_ of the y values that going into each graphical component that constructs his image. OR to push, translate + pop before and after creating him graphically. I'd recommend the first, so that you can see the process. It does sound cumbersome, but this is what happens. Review the Shiffman texts and code if you are unfamiliar with this. 

It'd also be great to see custom functions that pass parameters to make different instances of the functions. This could be done with reference to his floating. Perhaps under certain conditions he floats higher, lower, quivers, faster, slower, etc. If there is a variable attached to all the y values in a custom function, you can pass in values to then pertain to that y value. Then you can make different things that the user does in order to trigger these reactions. Mouse (speed, placement, how long in certain sections, etc) and keyboard would be fun.

od397 committed:
OrlandoDiaz_Project_1_Adjective

This is my finished version. With a misleading cursor following your
mouse to trick you into clicking (fitting with the “Untrustworthy”
adjective), once clicked Bill Cipher turns into his angry form from the
show, if you press “i/I” clones of Bill flash on the screen for a split
second (like a jumpscare!), if you hold down the mouse and move it then
a swarm of “batlike” bow ties fly across the screen, and holding down
and dragging will also make Bill’s Eyeball, Bowties, and Hat flash
random colors (similar to how he acts on the show). Depending on how
“active” your mouse is while dragging, the colors and bow ties will
either decrease or increase in speed and frequency respectively. I hope
this is good enough Katherine! :)
